_pages/saml.md

```diff
@@ -26,6 +26,8 @@ sidenav:
         href: "#logout-request"
       - text: Logout response
         href: "#logout-response"
+      - text: Remote (back-channel) logout
```

Comment:
style: I don't think backchannel is hyphenated? when I search online, I see "back channel" as a noun, but as a verb it's "backchanneled" no space, ex: https://en.wikipedia.org/wiki/Backchannel_(linguistics)

```suggestion
      - text: Remote (backchannel) logout
```

Reply:
Thanks! Apparently I've been doing it wrong the whole time 😛

Reply:
After chatting on Slack we're going to go with a space instead of all one word.